lib/LaTeXML/Common/Font/Metric.pm

```diff
@@ -152,12 +153,15 @@ sub process_lig_kern {
       $prognum = 256 * $op + $remainder; $firstinstr = 0;
       next; }
     $next = $$fontmap[$next];
+    $next = $$next[0] if ref $next;
```

Comment:
The changes handling dual type between arrayref vs scalar happen a bit often in this subroutine. This will be somewhat painful to port to Rust, but also makes me wonder why both are allowed in the first place.

Something that will be on my plate a little later, jsut noting it here.

Reply:
Indeed, there's a few such changes that leaked into this PR, but are really about the upcoming one. The idea is for the FontMap to store not only the equivalent Unicode, but some set of properties/attributes (esp. math related ones).  Ie. all still up for discussion.